sharding/shuffledOutTrigger.go

```diff
@@ -5,10 +5,14 @@ import (
 	"sync"
 
 	"github.com/ElrondNetwork/elrond-go-core/data/endProcess"
+	logger "github.com/ElrondNetwork/elrond-go-logger"
 	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/sharding/nodesCoordinator"
 )
 
-var _ ShuffledOutHandler = (*shuffledOutTrigger)(nil)
+var log = logger.GetOrCreate("sharding/shuffledOutTrigger")
```

Comment:
is this `log` only used in shuffleOutTrigger?
otherwise you should leave it for the package, so : `var log = logger.GetOrCreate("sharding")`

Reply:
yea, the log is used only in shuffleOutTriffer